src/libraries/System.Private.CoreLib/src/System/Threading/Mutex.Unix.cs

```diff
@@ -12,10 +12,15 @@ public sealed partial class Mutex
     {
         private void CreateMutexCore(bool initiallyOwned, string? name, out bool createdNew)
         {
-            // See https://github.com/dotnet/runtime/issues/48720
             if (name != null)
             {
-                throw new PlatformNotSupportedException(SR.PlatformNotSupported_NamedSynchronizationPrimitives);
+                var safeWaitHandle = WaitSubsystem.CreateNamedMutex(initiallyOwned, name, out createdNew);
```

Comment:
```suggestion
                SafeWaitHandle? safeWaitHandle = WaitSubsystem.CreateNamedMutex(initiallyOwned, name, out createdNew);
```
(multiple places) The repo coding conventions allow `var` only when the type is apparent from the right side. https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

Reply:
Ah, I wasn't aware of this, thanks.  Patch updated.